app/src/main/java/it/quip/android/actvitiy/CreateQuipActivity.java

```diff
@@ -0,0 +1,104 @@
+package it.quip.android.actvitiy;
+
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import java.util.List;
+
+import it.quip.android.QuipitApplication;
+import it.quip.android.R;
+import it.quip.android.fragment.CreateQuipCircleSelectFragment;
+import it.quip.android.fragment.CreateQuipComposeFragment;
+import it.quip.android.model.Circle;
+import it.quip.android.model.Quip;
+import it.quip.android.model.User;
+import it.quip.android.util.TimeUtils;
+
+public class CreateQuipActivity
+        extends AppCompatActivity
+        implements CreateQuipComposeFragment.OnComposeQuipListener {
+
+    private static final String COMPOSE = "compose_fragment";
+
+    private CreateQuipComposeFragment mCreateQuipComposeFragment;
```

Comment:
I added the Create prefix to make it easier to identify that CreateQuipComposeFragment and CreateQuipShareFragment are tied together, but I'm not tied to the idea. I can remove it.